src/include/btree_cmp_inline.h

```diff
@@ -111,13 +120,43 @@ __wt_compare(WT_SESSION_IMPL *session, WT_COLLATOR *collator, const WT_ITEM *use
     return (collator->compare(collator, &session->iface, user_item, tree_item, cmpp));
 }
 
+/*
+ * __check_fixed_string --
+ *     Check if the schema format is a fixed string.
+ */
+static inline int
+__check_fixed_string(const char *format)
```

Comment:
Will be moving this to a more appropriate place, if we decide to use this method to check for fixed string assertions